### DISCUSSION TITLE
Bearer token

### DISCUSSION THREAD

**Discussion by qawechi** [masterking32/MasterHamsterKombatBot#331] [Q&A] [answered]
Are there anywayz to get a bearer token of telegram with android?

**Comment by Lastnet0709** [comment, marked as answer]
You can try it with Kiwi Browser and follow the video tutorial from MasterKinG
[Top Free Hamster Combat Bot](https://youtu.be/cjWE7DmMFgw?t=205)

Or 

You can do the same as in the video after installing Violentmonkey and the script in the Kiwi browser.

https://github.com/user-attachments/assets/1ec8ecbb-783a-4b91-90bc-ecf39be5bbf2

**Reply by 8gprotein** [reply]
For whatever reason I can only login the hamster app on my android telegram app. I will be stuck on the loading page on desktop/web telegram app or android kiwi telegram web. Guess this is tough to get the bearer token.

**Comment by copycatcodex** [comment]
https://github.com/masterking32/MasterHamsterKombatBot/discussions/218#discussioncomment-10504591
i found in disscus
